conf/modules.config

```diff
@@ -353,34 +363,68 @@ if (params.remove_ribo_rna) {
 
 if (!params.skip_alignment) {
     process {
-        withName: 'NFCORE_RIBOSEQ:RIBOSEQ:.*:BAM_SORT_STATS_SAMTOOLS:BAM_STATS_SAMTOOLS:.*' {
-            ext.prefix = { "${meta.id}.sorted.bam" }
+
+        withName: 'NFCORE_RIBOSEQ:RIBOSEQ:.*:BAM_SORT_STATS_SAMTOOLS_GENOME:BAM_STATS_SAMTOOLS:.*' {
+            ext.prefix = { "${meta.id}.genome.sorted.bam" }
             publishDir = [
-                path: { "${params.outdir}/${params.aligner}/samtools_stats" },
+                path: { "${params.outdir}/alignment/${params.aligner}/sorted/samtools_stats" },
                 mode: params.publish_dir_mode,
                 pattern: "*.{stats,flagstat,idxstats}"
             ]
         }
 
-        withName: 'NFCORE_RIBOSEQ:RIBOSEQ:.*:BAM_SORT_STATS_SAMTOOLS:SAMTOOLS_SORT' {
-            ext.prefix = { "${meta.id}.sorted" }
+        withName: 'NFCORE_RIBOSEQ:RIBOSEQ:.*:BAM_SORT_STATS_SAMTOOLS_TRANSCRIPTOME:BAM_STATS_SAMTOOLS:.*' {
+            ext.prefix = { "${meta.id}.transcriptome.sorted.bam" }
             publishDir = [
-                path: { "${params.outdir}/${params.aligner}" },
+                path: { "${params.outdir}/alignment/${params.aligner}/sorted/samtools_stats" },
+                mode: params.publish_dir_mode,
+                pattern: "*.{stats,flagstat,idxstats}"
+            ]
+        }
+
+        withName: 'NFCORE_RIBOSEQ:RIBOSEQ:.*:BAM_SORT_STATS_SAMTOOLS_GENOME:SAMTOOLS_SORT' {
+            ext.prefix = { "${meta.id}.genome.sorted" }
+            publishDir = [
+                path: { "${params.outdir}/alignment/${params.aligner}/sorted" },
                 mode: params.publish_dir_mode,
                 pattern: "*.bam",
-                saveAs: { ( ['star_salmon','hisat2'].contains(params.aligner) &&
+                saveAs: { ( ['star','hisat2'].contains(params.aligner) &&
```

Comment:
why hisat2 if star is the only possible aligner?

Reply:
Thanks!